build/scripts/migrateDApp.sh

```diff
@@ -34,8 +27,8 @@ export ETH_RPC_URL=$CENT_ETHEREUM_NODEURL
 export ETH_PASSWORD="/dev/null"
 export ETH_FROM="0x89b0a86583c4444acfd71b463e0d3c55ae1412a5"
 
-regAddr=$(dapp create "TestNFT" "$ANCHOR_ADDR")
+regAddr=$(dapp create "NFT" "CentNFT" "CentNFT")
```

Comment:
Once we have the bridge assets populated we will have to pass here the ASSET_ADDR, for now it is fine.

Reply:
yeap!